src/components/lobby/LobbyPage.js

```diff
@@ -162,9 +174,21 @@ export default function LobbyPage() {
             />
           </form>
         )}
-        <Typography variant="h5" className={classes.margin}>
-          Game starts when everyone is ready
-        </Typography>
+        <Typography>{`Total number of custom words: ${numOfCustomWords}`}</Typography>
+        {numOfCustomWords !== 1 ? (
```

Comment:
I also accidentally committed this instead of leaving a suggestion but this is to alter to make it more clear to the users that they cannot start the game if they only have 1 custom word

Reply:
Great, thanks for the feedback!